### PR TITLE
Add pronunciation note as a subheader

### PR DESCRIPTION
**Pull request by darakian**
See https://github.com/hpyproject/hpy/issues/192#issuecomment-812455372
for more

Not super familiar with restructured text, but I believe this adds a subtitle/subheader

### CODE REVIEW

File: pages/index.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 What is HPy?
 ============
+Pronounced /h/ - pie
```
Comment (hodgestar) [inline pages/index.rst:13]: The `-----` makes a subsection in ReStructured Text. I would recommend just making this sentence a new paragraph after the "HPy provides ..." one below.

**Review by hodgestar — CHANGES_REQUESTED** [review]
Thank you for making this PR. Currently the text is a sub-section heading -- I think it should just be a new paragraph in this section.

**Comment by darakian** [comment]
> Thank you for making this PR. Currently the text is a sub-section heading -- I think it should just be a new paragraph in this section.

How do you feel about putting it in the `More info` list? I felt like it didn't read well in the sentence just after the main heading.

**Comment by hodgestar** [comment]
Putting it in "More Info" sounds good. I was wondering what pronunciation notation you were using? Using IPA I would write HPy as `/eɪtʃ-paɪ/` (see http://ipa-reader.xyz/ for how this sounds), but I am a complete pronunciation notation n00b (or `/nu:b/` :).

**Comment by darakian** [comment]
> Putting it in "More Info" sounds good. 

👍 
> I was wondering what pronunciation notation you were using?

Haha, I'm not using any in particular. Just using the `/h/` to call out h sound. I can swap it over to IPA if you prefer :)

File: pages/index.rst [hunk 2]
```diff
@@ -90,6 +90,8 @@ More info
 
 - #hpy IRC channel on the Freenode_ server
 
+- HPy is pronounced /h/ - pie
```
Comment (hodgestar) [inline pages/index.rst:93]:
```suggestion
- HPy is pronounced */h/ - pie* (or using IPA notation: */eɪtʃ-paɪ/*)
```

File: pages/index.rst [hunk 3]
```diff
@@ -90,6 +90,8 @@ More info
 
 - #hpy IRC channel on the Freenode_ server
 
+- HPy is pronounced */h/ - pie* (or using IPA notation: */eɪtʃ-paɪ/*)
```
Comment (hodgestar) [inline pages/index.rst:93]:
```suggestion
- HPy is pronounced `/h/ - pie` (or using IPA notation: `/eɪtʃ-paɪ/`)
```

**Comment by hodgestar** [comment]
> Haha, I'm not using any in particular. Just using the `/h/` to call out h sound. I can swap it over to IPA if you prefer :)

Let's have both! I think your simpler version is way more understandable, and we can have IPA for those interested in a rigorous guide. :)

**Pull request merged**